syntax = "proto2";

package botCode;


message Ack {

  required int32 hasAck = 1;

}